### PR TITLE
README: update the multipath.conf detail for load-balance support

### PR DESCRIPTION
**Pull request by lxbsz**
Signed-off-by: Xiubo Li <xiubli@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,6 +200,8 @@ Below we set mapth in Active/Passive mode; Note currently Active/Active is not s
 
 Please add the below configuration at the end of /etc/multipath.conf file.
 # LIO iSCSI
+
+For the versions both with and without load-balance supporting:
```
Comment (pkalever) [inline README.md]: For both the versions with and without load-balance support:

File: README.md [hunk 2]
```diff
@@ -214,6 +216,21 @@ devices {
         }
 }
 
+For the versions only with load-balance supporting:
```
Comment (pkalever) [inline README.md]: For versions with load-balance support:

**Comment by lxbsz** [comment]
@pkalever  Addressed them all, thanks.

**Comment by pkalever** [comment]
@lxbsz merged, thanks!

**Pull request merged**